File: C# Fundamentals May 2019/02.Exercises - Intro and Basic Syntax/PHP/02.More Exercises/SortNumbers.php
<?php
//solving only if statements
$firstNumber = floatval(readline());
$secondNumber = floatval(readline());
$thirdNumber = floatval(readline());

$biggestNumber = $firstNumber;

if ($secondNumber > $biggestNumber) {
    $biggestNumber = $secondNumber;
}

if ($thirdNumber > $biggestNumber) {
    $biggestNumber = $thirdNumber;
}

$smallestNumber = $firstNumber;

if ($secondNumber < $smallestNumber) {
    $smallestNumber = $secondNumber;
}

if ($thirdNumber < $smallestNumber) {
    $smallestNumber = $thirdNumber;
}

$middleNumber = $firstNumber;

for ($i = 0; $i < 3; $i++) {
    if (!($middleNumber !== $biggestNumber && $middleNumber !== $smallestNumber)) {
        if ($i === 0) {
            $middleNumber = $secondNumber;
        } else {
            $middleNumber = $thirdNumber;
        }
    } else {
        break;
    }
}

echo "$biggestNumber".PHP_EOL;
echo "$middleNumber".PHP_EOL;
echo "$smallestNumber".PHP_EOL;

?>

File: C# Fundamentals May 2019/02.Exercises - Intro and Basic Syntax/Java/More Exercises/src/SortNumbers.java
import java.util.Scanner;

public class SortNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int firstNumber = Integer.parseInt(scanner.nextLine());
        int secondNumber = Integer.parseInt(scanner.nextLine());
        int thirdNumber = Integer.parseInt(scanner.nextLine());

        int biggestNumber = firstNumber;

        if (secondNumber > biggestNumber) {
            biggestNumber = secondNumber;
        }

        if (thirdNumber > biggestNumber) {
            biggestNumber = thirdNumber;
        }

        int smallestNumber = firstNumber;

        if (secondNumber < smallestNumber) {
            smallestNumber = secondNumber;
        }

        if (thirdNumber < smallestNumber) {
            smallestNumber = thirdNumber;
        }

        int middleNumber = firstNumber;

        boolean isFoundMiddleNumber = false;
        int counter = 1;

        while (!(isFoundMiddleNumber)) {
            if (counter == 1) {
                if (!(middleNumber > smallestNumber && middleNumber < biggestNumber)) {
                    middleNumber = secondNumber;
                }
            } else {
                if (!(middleNumber > smallestNumber && middleNumber < biggestNumber)) {
                    middleNumber = thirdNumber;
                }
            }
            counter++;
            isFoundMiddleNumber = counter >= 3;
        }

        System.out.println(biggestNumber);
        System.out.println(middleNumber);
        System.out.println(smallestNumber);
    }
}
